ports/cosmiccleaning/port.json

```
@@ -0,0 +1,28 @@
+{
+  "version": 3,
+  "name": "cosmiccleaning.zip",
+  "items": [
+    "Cosmic Cleaning Crew.sh",
+    "cosmiccleaning"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Cosmic Cleaning Crew",
+    "porter": [
+      "Ganimoth"
+    ],
+    "desc": "3000 AD: Earth is polluted to the point of being uninhabitable. Humans have constructed rockets to convey the populace to Mars. Catastrophe! The heavens have been crammed with trash, barring all escape. It is up to you, the Galactic Garbage Man, to save all humanity! This game was created for the GBJam 11.",
+    "desc_md": null,
+    "inst": "Ready to run.",
+    "inst_md": null,
+    "genres": [
+      "puzzle"
+    ],
+    "image": null,
+    "rtr": true,
+    "exp": false,
+    "runtime": "frt_4.0.4.squashfs",
+    "reqs": [],
+    "arch": []
```

I think it's aarch64 :)
```suggestion
    "arch": [
        "aarch64"
    ]
```

---

Well, I havnt seen architecture specified in json for any godot port yet, or actually any port using runtimes, it a new requirement?